### PR TITLE
feat(curriculum): add animations to block 3.3 of the English Curriculum

### PR DESCRIPTION
**Pull request by DanielRosa74**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d750f42fb8c69d48edb565.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d750f42fb8c69d48edb565.md:1]: Add Maria's sentence: He's been with us for some time, nearly ten years. He's tall and has curly hair. He always has this vigilant look, probably a result of his job.

I also suggest adding some gaps to the fill-in-the-blank to cover these new sentences.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d75185d536899fd3c01977.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d75185d536899fd3c01977.md:1]: Add to 62 and 63: He sounds like a dedicated professional. I've always wondered what the folks in Security do around here. What are his responsibilities?

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d87217064c730ef7bc63fe.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d87217064c730ef7bc63fe.md:1]: Add space between Maria and Tom's lines.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d890f37666763b1c08e284.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d890f37666763b1c08e284.md:1]: this one doesn't need Bob's line

**Review by nieldakarla — CHANGES_REQUESTED** [review]
That is excellent, @DanielRosa74 ! I've added a few comments on 3 tasks that need to be reordered and missing sentences that weren't covered. 

I also suggest you include the missing sentences to the text of type 22 tasks. Maybe even adding a few blanks if necessary.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d892ad7262d64a5db56906.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d892ad7262d64a5db56906.md:1]: Task 78 one doesn't need Bob's line

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d8713fd64b650c269676cd.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d8713fd64b650c269676cd.md:1]: Add space between Maria and Tom's lines.

**Review by nieldakarla — APPROVED** [review]

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d8713fd64b650c269676cd.md
Comment (DanielRosa74) [reply curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d8713fd64b650c269676cd.md:1]: A new line has no effect on the template literals and, if I got it correctly, there can be only one fill-in-the-blank line.

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**